odoo-shippable/scripts/build-image.sh

```diff
@@ -121,14 +124,19 @@ git_clone_execute "${OH_MY_ZSH_REPO}" "master" "tools/install.sh"
 git_clone_copy "${ZSH_THEME_REPO}" "master" "schminitz.zsh-theme" "~/.oh-my-zsh/themes/odoo-shippable.zsh-theme"
 sed -i 's/robbyrussell/odoo-shippable/g' ~/.zshrc
 
-# Install & configure vim
+# Upgrade & configure vim
+apt-get upgrade vim
+wget -q -O /usr/share/vim/vim74/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/runtime/spell/es.utf-8.spl
 git_clone_execute "${SPF13_REPO}" "3.0" "bootstrap.sh"
 git_clone_copy "${VIM_OPENERP_REPO}" "master" "vim/" "~/.vim/bundle/vim-openerp"
-wget -q -O /usr/share/vim/vim74/spell/es.utf-8.spl http://ftp.vim.org/pub/vim/runtime/spell/es.utf-8.spl
 
 sed -i 's/ set mouse\=a/\"set mouse\=a/g' ~/.vimrc
 sed -i "s/let g:neocomplete#enable_at_startup = 1/let g:neocomplete#enable_at_startup = 0/g" ~/.vimrc
 
+# Install YouCompleteMe
+git clone https://github.com/Valloric/YouCompleteMe.git ~/.vim/bundle/YouCompleteMe
+(cd ~/.vim/bundle/YouCompleteMe && git submodule update --init --recursive && ./install.py)
```

Comment:
@JesusZapata 
Could you check if using `git_clone_execute` method from `library.sh` works?

I means, something like as:
`VIM_YOUCOMPLETEME_REPO="https://github.com/Valloric/YouCompleteMe.git"`
`git_clone_execute ${VIM_YOUCOMPLETEME_REPO} "master" "install.py"`

NOTE: This is just a example